src/py/flwr/server/run_serverapp.py

```diff
@@ -169,6 +175,8 @@ def run_server_app() -> None:  # pylint: disable=too-many-branches
         # Overwrite driver._run_id
         driver._run_id = res.run_id  # pylint: disable=W0212
 
+    server_config = {}
```

Comment:
Let's refer to this as `run_config` everywhere:
```suggestion
    run_config = {}
```